employees/templates/employees/partial/projects_work_hours_per_author.html

```diff
@@ -0,0 +1,26 @@
+{% load data_structure_element_selectors %}
+{% load data_display_filters %}
+<table class="table table-responsive-sm">
```

Comment:
Please leave blank line between imports and rest of html

Reply:
Done